app/root.tsx

```diff
@@ -42,6 +43,13 @@ export const meta: MetaFunction = () => ({
 });
 
 export default function App() {
+
+  const matches = useMatches();
+
+  const includeScripts = matches.some(
+    (match) => match.handle?.hydrate
+  );
```

Comment:
Ah nice, where'd you find this trick?

Reply:
I found it in the [remix docs](https://remix.run/docs/en/v1/guides/disabling-javascript).

Reply:
:)